shared/xccdf/system/permissions/files.xml

```diff
@@ -111,6 +122,18 @@ is critical for system security.</rationale>
 <ref nist="AC-6" disa=""/>
 </Rule>
 
+<Rule id="file_groupowner_etc_gshadow" severity="medium" prodtype="debian8,ubuntu1404">
```

Comment:
Nack. This rule should be merged into the existing rule, and `prodtype` added to `<description />` and `<ocil />` elements of the existing rule.